plumgrid_playbooks/roles/create_ifcs/tasks/main.yml

```diff
@@ -14,6 +14,16 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+- name: Update PLUMgrid Data Directory
+  delegate_to: localhost
+  replace: 
+    dest: /opt/plumgrid-ansible-standalone/plumgrid_playbooks/roles/create_ifcs/files/helpers.sh
+    regexp: "{{ item.regexp}}"
+    replace: "{{ item.replace}}"
+  with_items:
+    - { regexp: '^plumgrid_data_dir=.*', replace: 'plumgrid_data_dir="{{plumgrid_data_dir}}"' }
```

Comment:
too much redirection going on here, try to pass the directory paths to this script directly from the variable instead of replacing values in files